libunwind/src/UnwindRegistersSave.S

```diff
@@ -1273,6 +1273,19 @@ DEFINE_LIBUNWIND_FUNCTION(__unw_getcontext)
   .endr
 # endif
 
+#ifdef _LIBUNWIND_HAS_CHERI_LIB_C18N
+  // Store the trusted stack pointer
+  caddi     csp, csp, -32
+  sc        ca0, 0(csp)
+  sc        cra, 16(csp)
```

Comment:
Can we use `__SIZEOF_CHERI_CAPABILITY__` here (and for the frame sizes)? We don't have an RV32 platform for this but there's no reason not to make it work when it's this easy (and you've already used `__SIZEOF_CHERI_CAPABILITY__` for the final store, and for the restore path).